graphql/query_resolver.go

```diff
@@ -529,7 +534,13 @@ func (r *queryResolver) TaskAllExecutions(ctx context.Context, taskID string) ([
 			return nil, ResourceNotFound.Send(ctx, err.Error())
 		}
 		if dbTask == nil {
-			return nil, werrors.Errorf("unable to find task %s", taskID)
+			// If not found, grab the latest execution and display those results
```

Comment:
I added some functionality in case of an error, but this function is called for the dropdown info for an execution task/display task/task. So my thinking was, when the latest execution wasn't found, or a specific execution wasn't found, its result is cached from the last execution. This accounts for if the latest also grabbed their results from their previous.